NextcloudTalk/NCPeerConnection.m

```diff
@@ -379,13 +379,9 @@ - (void)peerConnection:(RTCPeerConnection *)peerConnection didRemoveIceCandidate
 - (void)peerConnection:(RTCPeerConnection *)peerConnection didOpenDataChannel:(RTCDataChannel *)dataChannel
 {
     [[WebRTCCommon shared] dispatch:^{
-        if ([dataChannel.label isEqualToString:@"status"] || [dataChannel.label isEqualToString:@"JanusDataChannel"]) {
-            self->_remoteDataChannel = dataChannel;
-            self->_remoteDataChannel.delegate = self;
-            NSLog(@"Remote data channel '%@' was opened.", dataChannel.label);
-        } else {
-            NSLog(@"Data channel '%@' was opened.", dataChannel.label);
-        }
+        self->_remoteDataChannel = dataChannel;
```

Comment:
Maybe just to be sure, add a log statement if a data channel already exists?

Reply:
Good idea, thx